lib/membrane_opus/parser/delimitation.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Membrane.Opus.Parser.Delimitation do
+  @moduledoc false
+  # Helper module for delimiting or undelimiting packets
+
+  @type delimitation_t :: :delimit | :undelimit | :keep
+  @type handler_t :: __MODULE__.Undelimiter | __MODULE__.Delimiter | __MODULE__.Keeper
```

Comment:
is there any way to use behaviours with typespecs in a more general way? In other words, I can't seem to refer to any module that implements the `Handler` behaviour, as a type

Reply:
It seems that `@type handler_t :: module` is the best you can do. But you can ask on Elixir forum.